packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -977,14 +1059,13 @@ export class MgtPerson extends MgtTemplatedComponent {
     return (
       this.renderTemplate('person-card', { person: personDetails, personImage: image }) ||
       html`
-        <mgt-person-card
-          lock-tab-navigation
```

Comment:
AFAIK removing this property makes it so that when the user opens the detail card they can tab out of the detail card without closing it. 
This can result in focus transferring to another person in the list while they are not visible as they are obscured behind the open card.
Is this the behavior we want?